actions/websocket_actions.jsx

```diff
@@ -524,10 +525,36 @@ function handlePreferencesDeletedEvent(msg) {
 }
 
 function handleUserTypingEvent(msg) {
-    GlobalActions.emitRemoteUserTypingEvent(msg.broadcast.channel_id, msg.data.user_id, msg.data.parent_id);
+    const state = getState();
+    const {currentUserId, profiles, statuses} = state.entities.users;
```

Comment:
Worth considering `getCurrentUserId` and the like for these?

Reply:
Yes, I can try to use selectors there too. I'll do the changes.

Reply:
Done! :)